static/js/components/UIPreferences.jsx

```diff
@@ -13,6 +13,7 @@ const UIPreferences = () => {
   const currentTheme = preferences?.theme;
   const invertThumbnails = preferences?.invertThumbnails || false;
   const compactComments = preferences?.compactComments || false;
+  const twelveOrTwentyFour = preferences?.twelveOrTwentyFour || false;
```

Comment:
What are the values going to be? `true` or `false`? Which corresponds to 24h and which to am/pm?

Reply:
yes making this more obvious in the variable name now :)